app/assets/v2/js/grants/new.js

```diff
@@ -71,64 +71,39 @@ $(document).ready(function() {
           SubscriptionContract.deploy({
             data: compiledSubscription.bytecode,
             arguments: args
-          })
-            .send({
-              from: accounts[0],
-              gas: 2500000,
-              gasPrice: 4000000000
-            })
-            .on('error', function(error) {
-              console.log('1', error);
-            })
-            .on('transactionHash', function(transactionHash) {
-              console.log('2', transactionHash);
-              $('#transaction_hash').val(transactionHash);
-
-              // Waiting State screen
-              $('#new-grant').hide();
-              $('.interior .body').addClass('open');
-              $('.interior .body').addClass('loading');
-              $('.grant_waiting').show();
-              document.issueURL = $('#input-url').val();
-              waitingStateActive();
-
-            })
-            .on('receipt', function(receipt) {
-              $('#receipt').val(JSON.stringify(receipt));
-              $('#contract_address').val(receipt.contractAddress);
-            })
-            .then(function(contractInstance) {
-
-              console.log(contractInstance);
-
-              $.each($(form).serializeArray(), function() {
-                data[this.name] = this.value;
-              });
-              console.log(data);
-              form.submit();
+          }).send({
+            from: accounts[0],
+            gas: 2500000,
+            gasPrice: 4000000000
+          }).on('error', function(error) {
+            console.log('1', error);
+          }).on('transactionHash', function(transactionHash) {
+            console.log('2', transactionHash);
+            $('#transaction_hash').val(transactionHash);
+
+            // Waiting State screen
+            $('#new-grant').hide();
+            $('.interior .body').addClass('open');
+            $('.interior .body').addClass('loading');
+            $('.grant_waiting').show();
+            document.issueURL = $('#input-url').val();
+            waitingStateActive();
+          }).on('receipt', function(receipt) {
+            $('#receipt').val(JSON.stringify(receipt));
+            $('#contract_address').val(receipt.contractAddress);
+          }).then(function(contractInstance) {
+            console.log(contractInstance);
+            $.each($(form).serializeArray(), function() {
+              data[this.name] = this.value;
             });
+            console.log(data);
+            form.submit();
+          });
         });
       });
     }
   });
 
-  $('#new-milestone').on('click', function(event) {
```

Comment:
Why has this been deleted? Was it unused?

Reply:
yeah we don't allow adding `milestones` in new grant right ? 
This was added way back if I'm not mistaken

Reply:
Ah, yes, you are right. We'll likely eventually add back in milestones to grants/new, but we can delete it for now.